apps/site/src/app/globals.css

```diff
@@ -1,27 +1,30 @@
+@import url("https://fonts.googleapis.com/css2?family=Alegreya:wght@400;500;600;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Mulish:wght@400;500;600;700&display=swap");
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
 :root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+	--color-cream: #fffce2;
+	--color-brown: #432810;
```

Comment:
Note: not too familiar with Tailwind, but we can look into more native was to specify color themes.